config/config.toml

```diff
@@ -46,34 +46,36 @@ DepositSimulation = ["secp", "bls", "deposit_data", "events"]
 
 # Modify constants in Constants.java
 [constants]
+
 # Minimal preset
 
 
 # Misc
 # ---------------------------------------------------------------
-#
+
 # [customized] Just 8 shards for testing purposes
 SHARD_COUNT = 8
 # [customized] unsecure, but fast
 TARGET_COMMITTEE_SIZE = 4
 # 2**12 (= 4,096)
-MAX_INDICES_PER_ATTESTATION = 4096
+MAX_VALIDATORS_PER_COMMITTEE = 4096
 # 2**2 (= 4)
 MIN_PER_EPOCH_CHURN_LIMIT = 4
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT = 65536
-# Normalizes base rewards
-BASE_REWARDS_PER_EPOCH = 5
 # [customized] Faster, but unsecure.
 SHUFFLE_ROUND_COUNT = 10
+# [customized]
```

Comment:
This is your note indicating our const diverges from the spec, right?

Reply:
I copy pasted this from spec minimal.yaml